views/registration.hbs

```diff
@@ -36,18 +36,22 @@
 
         <div class="background theme-yellow col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-xs-10 col-xs-offset-1">
             <form class='ideahacks-form'>
+                <h1> REGISTER</h1>
+                <h2>Choose email and password</h2>
 
-                <p class='error-message'></p>
+                <div class="input-wrapper">
+                    <p class='error-message'></p>
 
-                <label for="username">Email</label>
-                <input type="email" name="email" class="form-control" id="username" placeholder="Enter an email" required>
+                    <label for="username">EMAIL</label>
```

Comment:
I was wondering if it'll be an easier fix to add text-transform: uppercase; to label in the .css file. Since in future designs the text for label may or may not be all uppercase.

Reply:
I agree, what you said makes a lot of sense. I've reverted it back to lowercase, thanks!